command/validator/registration/register_validator.go

```diff
@@ -103,23 +88,6 @@ func runCommand(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	if params.amountValue.Cmp(big.NewInt(0)) > 0 {
-		approveTxn, err := bridgeHelper.CreateApproveERC20Txn(params.amountValue,
```

Comment:
Bare in mind that even though you will be staking a fixed (hardcoded) amount of native tokens on the `StakeManager` when you register a validator, you still need to approve `StakeManager` to take that fixed amount of tokens from the validator address.